topics/variant-analysis/tutorials/dunovo/tutorial.md

```diff
@@ -104,23 +103,41 @@ The starting point of the analysis is sequencing reads (in [FASTQ](https://en.wi
 
 ## Getting data in and assessing quality
 
-We uploaded the [Schmitt *et al.* 2015](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4414912/) data directly from SRA as shown in [this screencast](https://vimeo.com/121187220).
+You can obtain the data from [Schmitt *et al.* 2015](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4414912/) from Zenodo, or just import the [Galaxy history](https://usegalaxy.org/u/nstoler/h/srr1799908---schmitt-2015) that accompanies this tutorial.
 
-> ### {% icon hands_on %} Hands-on: Importing the raw data
+> ### {% icon hands_on %} Hands-on: Importing the input FASTQ files
 >
-> You can obtain the reads from this dataset by copying [this history](https://usegalaxy.org/u/nstoler/h/srr1799908---schmitt-2015).
+> Option 1: Import the files from Zenodo (this will work with any Galaxy instance):
```

Comment:
Could you make this hands on box look more like this one? https://github.com/galaxyproject/training-material/blob/master/topics/sequence-analysis/tutorials/quality-control/tutorial.md#inspect-a-raw-sequence-file

Most of the upload steps across the training materials are quite uniform and it might help users to see similar things here.